macOS/DBPE2ETests/DBPEndToEndTests.swift

```diff
@@ -125,12 +125,16 @@ final class DBPEndToEndTests: XCTestCase {
         await awaitFulfillment(of: profileSavedExpectation,
                                withTimeout: 3,
                                whenCondition: {
-            try! database.fetchProfile() != nil
+            autoreleasepool {
```

Comment:
Can we add a comment explaining why this is necessary? Otherwise it's prone to getting deleted by someone in the future.

Side note, I have a vague memory that there was a better way to implement this using Swift Testing, but at the time it wasn't released yet. So in future we might be able to remove the hack

Reply:
comment added